### PR TITLE
webeu1 Samar Novina Sprint-Challenge--User-Interface

### PR DESCRIPTION
**Pull request by novinary**
@samarv

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -99,6 +130,107 @@ p {
     display: flex;
     flex-wrap: wrap;
     justify-content: space-evenly;
+    justify-content: center;
+    align-items: center;
+}
+
+.box-1 {
```
Comment (samarv) [inline css/index.css:143]: Don't repeat yourself. It's better to create a single Box class and use 
something like `.box:nth-child(1)`

File: css/index.css [hunk 2]
```diff
@@ -99,6 +130,107 @@ p {
...
+	width:100px;
+	height: 100px;
+	margin: 20px 3.5%;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	background: teal;
+}
+.box-2 {
```
Comment (samarv) [inline css/index.css:152]: Again, Keep it DRY

File: css/index.css [hunk 3]
```diff
@@ -99,6 +130,107 @@ p {
...
+	width:100px;
+	height: 100px;
+	margin: 20px 3.5%;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	background: gold;
+}
+
+.box-3 {
```
Comment (samarv) [inline css/index.css:162]: Again, Keep it DRY

File: css/index.css [hunk 4]
```diff
@@ -99,6 +130,107 @@ p {
...
+	width:100px;
+	height: 100px;
+	margin: 20px 3.5%;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	background: cadetblue;
+}
+
+.box-4 {
```
Comment (samarv) [inline css/index.css:172]: Again, Keep it DRY

File: css/index.css [hunk 5]
```diff
@@ -99,6 +130,107 @@ p {
...
+	width:100px;
+	height: 100px;
+	margin: 20px 3.5%;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	background: coral;
+}
+
+.box-5 {
```
Comment (samarv) [inline css/index.css:182]: Again, Keep it DRY

File: css/index.css [hunk 6]
```diff
@@ -99,6 +130,107 @@ p {
...
+	width:100px;
+	height: 100px;
+	margin: 20px 3.5%;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	background: crimson;
+}
+
+.box-6 {
```
Comment (samarv) [inline css/index.css:192]: okay, one last time -just to make sure you remember - Keep it DRY

File: about.html [hunk 7]
```diff
@@ -3,62 +3,70 @@
...
+          <a href="#" class="products">Products</a>
+          <a href="#" class="blog">Blog</a>
+          <a href="#" class="contacts">Contacts</a>
+        </nav>
+    </header>  
+    <img class=jumbo src="img/jumbo-about.png" alt="city">
+
+    <h4>About</h4>
+          Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+          <hr>
```
Comment (samarv) [inline about.html:27]: `<hr>` is Dotted in Design file. You can use something like

`hr {
    border-top:1px dotted black;
}`

File: about.html [hunk 8]
```diff
@@ -3,62 +3,70 @@
...
+
+        <div class="mid-section">
+          <div class="mid-content">
+            <img src="img/about-plan.png" alt="strategy">
+          <h4>Strategy</h4>
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+            <button>Learn More</button>
```
Comment (samarv) [inline about.html:34]: Try to style buttons so that it matches design file

File: about.html [hunk 9]
```diff
@@ -3,62 +3,70 @@
...
+          <h4>Let's Work Together</h4>
+          <p>
+            Leverage agile frameworks to provide a robust synopsis for high level overviews.
+        Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+        Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+      </p>
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
       <footer>
```
Comment (samarv) [inline about.html:70]: Add some margin bottom on footer.

**Review by samarv — APPROVED** [review]
## Overview
Novina. Your overall code is clean and to the point. The overall design is close to the design file but it seems like you didn't style the buttons and Horizontal ruler. The only other things that i will say is that you may need to think about your commits. so you may want to consider a more granular git commit workflow that gives clear commits for the goals that were required. so about 1 commit per sub task. and maybe adding (MVP) to the front of MVP features and (STRETCH) to the front of STRETCH features.

**_Breakdown_**

**MVP**

* [X] Build the HTML and CSS to create the missing navigation and header.
* [X] Link the `About` navigation item to the [about.html](about.html) page
* [X] Color Boxes
* [X] Copy and paste your home page navigation and header into the about page
* [X] Update the header image with the about page image
* [X] Link the `Home` navigation item back to the `index.html` page.
* [X] Build the rest of the about page layout to match the design

**Stretch**

* [ ] Build a page of your choosing from the navigation items.  Come up with content and images that fit the theme.  
* [ ] Introduce CSS animations to your site.
* [ ] Build a contact page and create a form with several inputs of your choosing
* [ ] Add responsive breakpoints to your code by using media queries


**Conclusion** 


Novina, You did well and you have made good progress in last few days. It's good to see you get the hang of CSS and Flexbox. Website is pretty close to Design file.  It would be awesome, If you can attempt stretch as well!  I am really looking forward to seeing more work from you. and I think that if you keep up with this quality of work across the board you will do fine here at lambda 👍 *remember that if any doubt about process or procedure we are always available in the help channel and feel free to reach out to me for any further explanations or clarifications of anything said in this review*

File: about.html (hunk 7 above)
Comment (novinary) [reply about.html:27]: I have now added Dashed border property to hr element

File: about.html (hunk 8 above)
Comment (novinary) [reply about.html:34]: I have now added a style to buttons to match the design file

**Pull request merged**